tests/integration/individual_response/test_individual_response.py

```diff
@@ -1,1336 +0,0 @@
-# pylint: disable=too-many-lines
```

Comment:
You need to also remove: 
```js
individualResponseGuidance() {
    return '[data-qa="individual-response-url"]';
  }
```
from question.page.js

Reply:
Removed 👍